clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -6029,6 +6037,7 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
       CollectArgsForIntegratedAssembler(C, Args, CmdArgs, D);
     }
     if (IsSYCLDevice && IsSYCLNativeCPU) {
+      CmdArgs.push_back("-Wno-override-module");
```

Comment:
What is this addition for?  I've also noticed there is no driver test verifying the additional options passed during the device compile - can one be added?

Reply:
This is because the initial compilation for NativeCPU generates a module for an unknown target, that the second target-specific compilation then issues a warning on unless suppressed. Good idea to add a test for that, I will see how we can add one, thanks.

Reply:
We have `clang/test/Driver/sycl-native-cpu.cpp` already where I could add checks for that, done.